VKapp/VKapp/Extensions/UIViewController+Extension.swift

```diff
@@ -0,0 +1,26 @@
+// UIViewController+Extension.swift
```

Comment:
```suggestion
// LoginViewController+Extension.swift
```
Назови файл правильно